layout/common/head.jsx

```diff
@@ -107,14 +107,14 @@ module.exports = class extends Component {
             <title>{getPageTitle(page, config.title, helper)}</title>
 
             {typeof open_graph === 'object' ? <OpenGraph
-                type={open_graph.type || (is_post(page) ? 'article' : 'website')}
-                title={open_graph.title || page.title || config.title}
+                type={(is_post(page) ? 'article' : 'website') || open_graph.type}
```

Comment:
sometimes, both of them exist and it is better to choose the post.

Reply:
`open_graph` is set explicitly in _config.yml. Even if it appears magically somehow, you can use `open_graph: false` in the post's front-matter to disable it. However, it does not work the other way around since `is_post(post) ? A : B` will always return A or B and `open_graph.type` will never be used. This is also the case for `open_graph.title`. In order to set the title of open graph if it's different from the post's title, a user has to remove post's title first before letting `open_graph.title` take effect if you put `open_graph` behind.

Reply:
Yes,I see. You are right.Thanks again for your explanation.